libs/LibCSA/LibCSA.lua

```diff
@@ -115,11 +119,27 @@ local function Load()
 		end
 	end
 
+	ZO_CenterScreenAnnouncementCountdownLine_PlayCountdownLoopAnimation = ZO_CenterScreenAnnouncementCountdownLine.PlayCountdownLoopAnimation
+	ZO_CenterScreenAnnouncementCountdownLine.PlayCountdownLoopAnimation = function(self)
+		self.countdownControl:SetText(self.currentCountdownTimeS)
```

Comment:
In near future i want to implement custom text instead of 0.

Reply:
Yea I want to do that too, together with a "Stopped" text.

Reply:
```
if self.currentCountdownTimeS == 0 then
    self.countdownControl:SetText(self.customText)
else
    self.countdownControl:SetText(self.currentCountdownTimeS)
end
```
at 0 display for example "DODGE" or something but this should be inserted as additional text to StartCountdown(....., endText) or something like that so we can set this for every countdown.

Reply:
Ah you already thinkig about that. I thought you r asking what i want to do here.